samples/excel/85-preview-apis/range-find.yaml

```diff
@@ -0,0 +1,171 @@
+order: 3
+id: excel-range-find
+name: Find
+description: Finds data within a range.
+host: EXCEL
+api_set:
+    ExcelAPI: 1.9
+script:
+    content: |-
+        $("#setup").click(() => tryCatch(setup));
+        $("#findText").click(() => tryCatch(findText));
+        $("#findTextWithNullCheck").click(() => tryCatch(findTextWithNullCheck));
+        $("#toggleComplete").click(() => tryCatch(toggleComplete));
+        $("#toggleCase").click(() => tryCatch(toggleCase));
+        $("#toggleDirection").click(() => tryCatch(toggleDirection));
+
+        let completeMatch  = false;
+        let matchCase = false;
+        let searchDirection = Excel.SearchDirection.forward;
+
+        async function findText() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const table = sheet.tables.getItem("ExpensesTable");
+                const searchRange = table.getRange();
+                const foundRange = searchRange.find($("#searchText").text(), {
+                    completeMatch: completeMatch,
+                    matchCase: matchCase,
+                    searchDirection: searchDirection
+                });
+                
+                foundRange.load("address");
```

Comment:
Consider adding a code comment before this line to indicate that an error will be thrown if no match is found. This'll help the user to understand that the ItemNotFound error is by design (not a bug with the snippet). Maybe something along the lines of: 

`// Note: If no match was found, an ItemNotFound error will be thrown when this line attempts to load the address property.`